client/drawtext.lua

```diff
@@ -0,0 +1,60 @@
+local function hideText()
+    SendNUIMessage({
+        action = 'HIDE_TEXT',
+    })
+end
+
+local function drawText(text, position)
+    if not type(position) == "string" then position = "left" end
+
+    SendNUIMessage({
+        action = 'DRAW_TEXT',
+        data = {
+            text = text,
+            position = position
+        }
+    })
+end
+
+local function changeText(text, position)
+    if not type(position) == "string" then position = "left" end
+
+    SendNUIMessage({
+        action = 'CHANGE_TEXT',
+        data = {
+            text = text,
+            position = position
+        }
+    })
+end
+
+local function keyPressed()
+    CreateThread(function() -- Not sure if a thread is needed but why not eh?
```

Comment:
Doesn't this start a thread for each key pressed? I don't think this is needed. I tested it locally and it works fine without it.

Reply:
That's not how that works

Reply:
But yeah it's unnecessary